From: Isaac Hermida <isaac.hermida@digi.com>
Date: Tue, 5 Jul 2016 17:52:44 +0200
Subject: [PATCH] Kbuild: Group most of the relevant DEBUG options

The value of BUILD_DEBUG_VERSION will be used to compile the driver with most
of the relevant DEBUG flags. Use that variable to compile the debug version.

Signed-off-by: Isaac Hermida <isaac.hermida@digi.com>
---
 Kbuild | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/Kbuild b/Kbuild
index a0e203ce5493..e5f2e8ebd4ad 100644
--- a/Kbuild
+++ b/Kbuild
@@ -896,9 +896,7 @@ CDEFINES :=	-DANI_LITTLE_BYTE_ENDIAN \
 		-DWLAN_FEATURE_PACKET_FILTERING \
 		-DWLAN_FEATURE_VOWIFI \
 		-DWLAN_FEATURE_11AC \
-		-DWLAN_FEATURE_P2P_DEBUG \
 		-DWLAN_ENABLE_AGEIE_ON_SCAN_RESULTS \
-		-DWLANTL_DEBUG\
 		-DWLAN_NS_OFFLOAD \
 		-DWLAN_ACTIVEMODE_OFFLOAD_FEATURE \
 		-DWLAN_FEATURE_HOLD_RX_WAKELOCK \
@@ -936,8 +934,7 @@ CDEFINES +=     -DCONFIG_HL_SUPPORT \
                 -DSDIO_3_0 \
                 -DHIF_SDIO \
                 -DCONFIG_ATH_PROCFS_DIAG_SUPPORT \
-                -DFEATURE_HL_GROUP_CREDIT_FLOW_CONTROL \
-		-DDEBUG_HL_LOGGING
+                -DFEATURE_HL_GROUP_CREDIT_FLOW_CONTROL
 endif

 ifeq ($(CONFIG_QCA_WIFI_SDIO), 1)
@@ -974,8 +971,11 @@ CDEFINES += 	-DTRACE_RECORD \

 ifeq ($(BUILD_DEBUG_VERSION),1)
 CDEFINES +=	-DWLAN_DEBUG \
+		-DWLAN_FEATURE_P2P_DEBUG \
+		-DWLANTL_DEBUG\
 		-DPE_DEBUG_LOGW \
 		-DPE_DEBUG_LOGE \
+		-DDEBUG_HL_LOGGING \
 		-DDEBUG
 endif

